src/App.js

```diff
@@ -1,26 +1,104 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import React from "react";
+import logo from "./logo.svg";
+import "./App.css";
+import BookList from "./lib/BookList";
+import * as BooksAPI from "./BooksAPI";
+import Heading from "./lib/Heading";
+import SubHeading from "./lib/Subheading";
+
+const optionList = [
+  { label: "Reading", value: "reading" },
+  { label: "Want to Read", value: "wantToRead" },
+  { label: "Done", value: "done" }
+];
+
+const data = [
```

Comment:
Are we using this data anywhere? If no, it should be removed. Try to add AirBnb linter on and also active the linter on VScode. It should warn you when there are unused vars